src/richie/plugins/large_banner/templates/richie/large_banner/large_banner.html

```diff
@@ -10,6 +10,11 @@
                     {% thumbnail instance.background_image 768x450 crop=',0' %} 768w"
             alt="banner"/>
     {% endif %}
-    <h1 class="large-banner__title">{{ instance.title }}</h1>
-    <img src="{% thumbnail instance.logo 200x100 crop='smart' %}" class="large-banner__logo" alt="{{ instance.logo_alt_text }}"/>
+    <div class="large-banner__body">
+        <h1 class="large-banner__title">{{ instance.title }}</h1>
+        <img src="{% thumbnail instance.logo 200x100 crop='smart' %}" class="large-banner__logo" alt="{{ instance.logo_alt_text }}"/>
+        {% if instance.content %}
+            <div class="large-banner__content">{{ instance.content|safe }}</div>
```

Comment:
Don't mark it safe since we are using a raw text plugin and don't accept formatting?

Reply:
we are now using a very simple ckeditor widget